src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  selectEye: Todo;
+  setSelectEye: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectEye, setSelectEye }) => {
+  const [loading, setLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(selectEye.userId).then(person => {
+      setUser(person);
+      setLoading(false);
+    });
+  });
```

Comment:
The `useEffect` hook is missing a dependency array. This means the effect will run after every render, which is not the intended behavior. To fix this, add `[selectEye.userId]` as the dependency array to ensure the effect only runs when `selectEye.userId` changes.